sdk/src/connections/target/adsd3500_sensor.cpp

```diff
@@ -815,6 +815,10 @@ aditof::Status Adsd3500Sensor::adsd3500_read_cmd(uint16_t cmd, uint16_t *data) {
 
     extCtrl.p_u8 = buf;
 
+    if (cmd == 0x0054 ||
+        cmd == 0x0055) //in case of temperature requests response time needed
+        usleep(90000);
```

Comment:
How did you determine the 90 ms value?

Reply:
Witch consecutive trying, still waiting for confirm from the firmware side

Reply:
ADSD3500 need to process the last frame that sensor is currently working on in order to create a response. This line has been replaced with a generic solution based on frames per second of the sensor